src/lib/dev-environment/dev-environment-lando.js

```diff
@@ -244,7 +244,7 @@ async function addHooks( app: App, lando: Lando ) {
 			registryResolvable = false;
 		}
 
-		data.opts.pull = registryResolvable && instanceData.pullAfter < Date.now();
+		data.opts.pull = registryResolvable && ( instanceData.pullAfter || 0 ) < Date.now();
```

Comment:
* `undefined < Date.now() === false`
  * `undefined > Date.now() === false`
  * `undefined <= Date.now() === false`
  * `undefined => Date.now() === false`

`undefined` gets coerced to `NaN`, and `NaN` is a special beast because it is not even equal to itself.

Reply:
thanks!